app/views/course/forum/forums/index.html.slim

```diff
@@ -0,0 +1,14 @@
+= page_header do
+  = new_button([current_course, :forum]) if can?(:create, Course::Forum)
+table.table.forum-list.table-hover
+  thead
+    tr
+      th
+      th = t('.forum')
+      th = t('.topics')
+      th = t('.posts')
+      th = t('.views')
+      th = t('.subscription')
+      th
+  tbody
+    = render partial: 'forum', collection:  @forums
```

Comment:
Does not work. This will search `course/forums/_forum` partial. Since my controller is under `course/forum`, the views should be in `course/forum` folder.